KuringApp/KuringApp/Department/DepartmentSelector.swift

```diff
@@ -52,28 +52,78 @@ struct DepartmentSelector: View {
     
     var body: some View {
         WithViewStore(self.store, observe: { $0 }) { viewStore in
-            List {
-                Section {
+            VStack {
+                HStack(alignment: .center, spacing: 10) {
+                    Text("대표 학과 선택")
+                        .font(.system(size: 18, weight: .bold))
+                        .foregroundStyle(Color.black)
+                }
+                .padding(.horizontal, 20)
+                .padding(.top, 24)
+                .padding(.bottom, 12)
+                .frame(width: 375, alignment: .leading)
+                
+                ScrollView {
+                    
                     ForEach(viewStore.addedDepartment) { department in
-                        Button {
-                            viewStore.send(.selectDepartment(id: department.id))
-                        } label: {
-                            Label(
-                                department.korName,
-                                systemImage: department == viewStore.currentDepartment
+                        HStack {
+                            Text(department.id)
+                                .font(.system(size: 16, weight: .medium))
+                                .foregroundColor(.black)
+                            Spacer()
+                            Image(
+                                systemName: department == viewStore.currentDepartment
                                 ? "checkmark.circle.fill"
                                 : "circle"
                             )
+                            .foregroundStyle(
+                                department == viewStore.currentDepartment
+                                ? Color(red: 0.24, green: 0.74, blue: 0.5)
+                                : Color.black.opacity(0.1)
+                            )
+                            .frame(width: 20, height: 20)
+                        }
+                        .padding(.horizontal, 20)
+                        .padding(.vertical, 12)
+                        .frame(width: 375, alignment: .center)
```

Comment:
```suggestion
                        .frame(maxWidth: infinity)
```

375 고정값을 넣는 것 보다는 디바이스 사이즈 고려한 형태가 더 좋을 것 같습니다.

Reply:
수정 완료했습니다~!